### PR TITLE
Add new page about keeping the toilets clean

### PR DESCRIPTION
**Pull request by heidar**
I just want to make it clear that this is not a big problem, and this prevents it from becoming one. :)

As the team grows, it becomes more of a social responsiblity to keep the
toilets clean.

This page sets some guidelines on some steps we can all take to keep the
toilets nice and clean.

### CODE REVIEW

**Comment by chrisblackburn** [comment]
I'm unsure if this is a feature creep, but it could be interesting to expand this to include post-bathroom hand washing tips. It's a shared responsibility to prevent the spread of hand-foot-mouth, hep. A etc.

**Comment by heidar** [comment]
> I'm unsure if this is a feature creep, but it could be interesting to expand this to include post-bathroom hand washing tips. It's a shared responsibility to prevent the spread of hand-foot-mouth, hep. A etc.

An excellent idea. I have added a section for this.

File: guides/office/toilet.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+No one is too good to put in some hard manual labour and clean the toilets when necessary. Let's keep the toilets sparkling clean, together.
+
+## Toilet tenet
+
+When using the toilets, please make sure you perform the following:
+
+ - Flush the toilet after use, half flush if you can to save water (and the planet)
+ - If any fecal matter remains inside the toilet bowl, please use the toilet brush to clean it
+ - If the fecal matter is outside the toilet bowl, please seek medical attention
```
Comment (DelphineGattaz) [inline guides/office/toilet.md]: This probably can't stay in here as it is.

**Comment by DelphineGattaz** [comment]
I've made one comment, I think one line should be removed. The rest looks good to me!

File: guides/office/toilet.md (hunk 1 above)
Comment (yaseminercan) [reply guides/office/toilet.md]: Agree, need to rewrite.  do we need toilet guides?

Comment (yaseminercan) [reply guides/office/toilet.md]: Can we remove the toilet guide please? :) I'll call it out on slack (if I have to). Unless it becomes a problem I would rather not have this guide.

**Comment by lukemorton** [comment]
Closed at request of Yas.

**Comment by craigjbass** [comment]
🤣

**Pull request closed without merging**